PPAK_CVS/Sources/Scenes/Onboarding/OnboardingVC.swift

```diff
@@ -0,0 +1,168 @@
+//
+//  OnboardingViewController.swift
+//  PPAK_CVS
+//
+//  Created by 김민지 on 2022/09/24.
+//
+
+import Foundation
+import UIKit
```

Comment:
UIKit 내부에 Foundation이 import되어있으므로 제거하는 것이 좋다고 생각합니다!

```suggestion
import UIKit
```

Reply:
오 UIKit에 Foundation이 포함되어 있는 줄 몰랐어요.!! 감사합니다 

FTUXStorage는 감지 및 판단하지 않습니다. 
오직 UserDefatuls와 관련된 일만 하는 얘입니다.
처음 들어온 유저인지 판단하는 부분은 SceneDelegate에서 합니다!

Reply:
2번 질문을 이제 봤네요ㅠ
그러네요 Common은 공통으로 사용하는 것들이니까 차라리 Constants로 빼는게 좋겠어요!

Reply:
> FTUXStorage는 감지 및 판단하지 않습니다. 오직 UserDefatuls와 관련된 일만 하는 얘입니다. 처음 들어온 유저인지 판단하는 부분은 SceneDelegate에서 합니다!

제가 오해할만한 소지로 질문을 드렸네요. ㅎㅎ `FTUXStorage`는  분명 UserDefaults로 bool값을 처리하는 객체라는 것은 이해했습니다만, 결국 SceneDelegate에서 `FTUXStorage`의 역할은 **첫 실행유무를 확인해주는 용도**로 사용되고 있고, `OnboardingVC`에서 액션에 따라 `FTUXStorage`를 이용하여 **이제부터는 첫 실행이 아님**을 저장하는 것처럼 보여지고 있어서요!